core/Utils/Icon.vala

```diff
@@ -62,7 +62,7 @@ public class Noise.Icon : Object {
                 // This only works if Build.ICON_DIR contains a sub-directory named "hicolor"
                 // containing all the fallback icons (possibly organized into sub-folders as well),
                 // or if the icons are immediate children of this directory.
-                _theme.append_search_path (Path.build_filename (Build.ICON_DIR));
+                _theme.append_search_path (Path.build_filename ("/usr/share/noise/icons"));
```

Comment:
`Build.ICON_DIR` should be working I think.

Reply:
yep, working on a python script for post install I'll get to it. not sure chucking the hardcoded paths into build.vala is going to cut it, but it works :D